scripts/build_powerplants.py

```diff
@@ -154,31 +163,31 @@ def replace_natural_gas_fueltype(df):
     n = pypsa.Network(snakemake.input.network)
     countries = snakemake.params.countries
 
-    ppl = (
-        pm.powerplants(from_url=True)
-        .powerplant.fill_missing_decommissioning_years()
-        .powerplant.convert_country_to_alpha2()
-        .query('Fueltype not in ["Solar", "Wind"] and Country in @countries')
-        .assign(Technology=replace_natural_gas_technology)
-        .assign(Fueltype=replace_natural_gas_fueltype)
-        .replace({"Solid Biomass": "Bioenergy", "Biogas": "Bioenergy"})
-    )
-
-    # Correct bioenergy for countries where possible
-    opsd = pm.data.OPSD_VRE().powerplant.convert_country_to_alpha2()
-    opsd = opsd.replace({"Solid Biomass": "Bioenergy", "Biogas": "Bioenergy"}).query(
-        'Country in @countries and Fueltype == "Bioenergy"'
-    )
-    opsd["Name"] = "Biomass"
-    available_countries = opsd.Country.unique()
-    ppl = ppl.query('not (Country in @available_countries and Fueltype == "Bioenergy")')
-    ppl = pd.concat([ppl, opsd])
-
```

Comment:
why are we deleting/removing this? custom ppl functionality should come on top of default insertion of powerplants, not replace it entirely?